pkg/conversion/mapi2capi/aws_test.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package mapi2capi
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	configv1 "github.com/openshift/api/config/v1"
+	machinebuilder "github.com/openshift/cluster-api-actuator-pkg/testutils/resourcebuilder/machine/v1beta1"
+
+	"k8s.io/utils/ptr"
+)
+
+var _ = Describe("mapi2capi AWS", func() {
+	// mapiProviderConfig := &mapiv1.AWSMachineProviderConfig{
```

Comment:
Could we put this in a `testdata` JSON/YAML file instead of a comment and a builder? I worry that this comment will get out of date fast.

Reply:
This was used in a test that I deleted, I just commented it out so that the compiler wouldn't complain while I was hacking.

I'll get rid of it I think since we aren't actually going to use it